data/tanchuma_text_results.txt

```diff
@@ -287,7 +287,7 @@ THE VOICE OF JACOB, because he speaks in an entreating strain —“Arise I pray
 זה שלש רגלים THESE THREE TIMES — By these words he suggested to him: You seek to root out a nation that celebrates three festivals (שלש רגלים) annually?! (Midrash Tanchuma, Balak 9).
 לו יש חרב בידי I WOULD THERE WERE A SWORD IN MY HAND — This utterance was a great shame for him in the sight of the princes: this man was going for the purpose of slaying a whole nation by his mouth, and for this animal he required a weapon! (cf. Midrash Tanchuma, Balak 9).
 כי ירט הדרך לנגדי BECAUSE THY WAY IS PERVERSE BEFORE ME — Our Rabbis, the Mishnaic Sages, explain it as an acrostic, יראה she feared when ראתה, she saw, and נטתה she turned aside (Shabbat 105a), because that the journey (הדרך) is לנגדי, is against me, that is to say, to anger me, to affront me. — But according to its literal meaning, it is equivalent to חרד הדרך לנגדי, the word ירט being associated in meaning with רטט, “rapid movement” (cf.  49:24). The phrase therefore signifies: for I have seen the man who is taking the journey (Balaam) that he hastened and hurried on the journey, which was taken to my anger and to my despite. It is an elliptical verse (the words בעל הדרך being omitted after ירט), just as (II Samuel 13:39) “And the soul of David longed” (the word נפש being omitted). Another explanation is: ירט is an expression denoting “being pleased”. Similar is, (Job 16:11) “Through the wicked ירטני” — i.e., He appeases and comforts me through the wicked, whilst indeed they do nought but annoy me (Midrash Tanchuma, Balak 9).
-ואותה החייתי AND IT WOULD I HAVE KEPT ALIVE — but now because it spoke and rebuked you andyou could not stand against its rebuke, — just as it is written, "and he said, No!" — I have killed it in order that people should not be able to say: This is the animal that silenced Balaam by its rebuke, so that he could not reply. For God has regard for human dignity. Similar is (Leviticus 20:16): “And thou shalt slay the woman and the animal (with which sin was committed)”, and similarly Leviticus 20:15): “And the animal shalt thou slay” (Midrash Tanchuma, Balak 9).
+ואותה החייתי AND IT WOULD I HAVE KEPT ALIVE — but now because it spoke and rebuked you and you could not stand against its rebuke, — just as it is written, "and he said, No!" — I have killed it in order that people should not be able to say: This is the animal that silenced Balaam by its rebuke, so that he could not reply. For God has regard for human dignity. Similar is (Leviticus 20:16): “And thou shalt slay the woman and the animal (with which sin was committed)”, and similarly Leviticus 20:15): “And the animal shalt thou slay” (Midrash Tanchuma, Balak 9).
```

Comment:
I'm curious about this file in general, `andyou` appears to be a typo, and thus hopefully one would be happy to accept the fix, but if it's canon to include the typo, then the file would belong in my ignore branch.